pkgs/by-name/de/deskreen/package.nix

```diff
@@ -0,0 +1,89 @@
+{ lib
+, fetchurl
+, stdenvNoCC
+, appimageTools
+, alsa-lib
+, at-spi2-atk
+, cairo
+, cups
+, dbus
+, expat
+, glib
+, glibc
+, gtk3
+, hicolor-icon-theme
+, libdrm
+, libxkbcommon
+, mesa
+, nspr
+, nss
+, pango
+, xorg
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "deskreen";
+  version = "2.0.4";
+
+  src = fetchurl {
+    url = "https://github.com/pavlobu/deskreen/releases/download/v${finalAttrs.version}/Deskreen-${finalAttrs.version}.AppImage";
+    hash = "sha256-0jI/mbXaXanY6ay2zn+dPWGvsqWRcF8aYHRvfGVsObE=";
+  };
+
+  deskreenUnwrapped = appimageTools.wrapType2 {
+    name = "deskreen";
+    src = finalAttrs.src;
+    extraPkgs = pkgs: with pkgs; [
```

Comment:
Why?

Reply:
It's not obvious what you're asking when you just point at a line and say *why*. Is it the `with` statement you're hinting at?

Reply:
Good suggestion. Thanks.
What it is advised to do in this case then? What do you advise?

Reply:
I know nothing about appimageTools, but looking at the Nixpkgs, I have seen many of  them that declare only lib, stdenv, fetchers and appimageTools in the input set; when they use extraPackages, they do not declare them on the input set.

So, my suggestion (with big grains of salt) would be remove them from input attrset.